projects/components/src/paginator/paginator.component.ts

```diff
@@ -217,7 +217,9 @@ export class PaginatorComponent implements OnChanges, PaginationProvider {
     }
 
     if (this.totalItems === PaginatorTotalCode.Last) {
-      return 'last';
+      const approxTotal = (this.pageIndex + 1) * this.pageSize;
```

Comment:
We should never guess, but this case shouldn't exist - we would want to remove this enum value entirely  and update all usages (as you did in traces table data source model).

Reply:
Removed the last enum value and swapped with exact total being passed from "projects/observability/src/shared/dashboard/data/graphql/table/traces/traces-table-data-source.model.ts" line no 64